OpenRA.Game/Server/Connection.cs

```diff
@@ -104,8 +104,10 @@ public void ReadData(Server server)
 
 						case ReceiveState.Data:
 							{
+								if (Frame > MostRecentFrame)
+									MostRecentFrame = Frame;
```

Comment:
I often like to keep the variable being assigned to on the left in the conditional in expressions like this, i.e.
```c#
if (MostRecentFrame < Frame)
    MostRecentFrame = Frame;
```

That's probably just a matter of taste though.